README.md

```diff
@@ -114,31 +114,34 @@ CDTDatastore *datastore = [manager datastoreNamed:@"my_datastore"
                                             error:&outError];
 
 // Create a document
-CDTMutableDocumentRevision *rev = [CDTMutableDocumentRevision revision];
-rev.docId = @"doc1";  // Or don't and get an ID generated for you
-rev.body = @{
+CDTDocumentRevision *rev = [CDTDocumentRevision revisionWithDocId:@"doc1"];
+// Use [CDTDocumentRevision revision] to get an ID generated for you on saving
+rev.body = @[@{
```

Comment:
Drop the first `@` because the opening square bracket is for a message invocation, not array literal